cns/hnsclient/hnsclient_windows.go

```diff
@@ -426,6 +433,31 @@ func configureAclSettingHostNCApipaEndpoint(
 				return nil, err
 			}
 		}
+
+	}
+
+	if ncRequestedPolicies != nil {
+		// Iterate thru the requested endpoint policies where policy type is ACL, endpoint type is APIPA
+		// include the raw json message in the endpoint policies
+		for _, requestedPolicy := range ncRequestedPolicies {
+
+			if strings.EqualFold(requestedPolicy.Type, aclPolicyType) && strings.EqualFold(requestedPolicy.EndpointType, apipaEndpointType) {
+
+				var requestedAclPolicy hcn.AclPolicySetting
+
+				if err := json.Unmarshal(requestedPolicy.Settings, &requestedAclPolicy); err != nil {
```

Comment:
I think you don't need to create new err here. ( err := )

Reply:
Changed